steps/steps-app-gallery-reports/0.9.0/step.yml

```diff
@@ -0,0 +1,245 @@
+title: Huawei AppGallery Connect Reports
+summary: Gets the different Reports from AppGallery Connect for a specific published
```

Comment:
In this context `various` should be used instead of `different`.

Reply:
I've just checked with a copyrighter and in that case different or various could match anyway. But I would change the whole sentence in the next version potentially